.github/wiki/Getting-Started.md

```diff
@@ -66,6 +66,15 @@ funcsave cd
 funcsave check_directory_for_new_repository
 ```
 
+By [**@mataha**](https://github.com/mataha)
```

Comment:
Minor (non-blocking) nitpick: we probably want all of these snippet credits to have the same style, so we don't show favor for one author over another. So maybe either update this line's style, or update the style for all of the other author credits.

Reply:
Done.